website/content/docs/getting-started/_index.md

```diff
@@ -4,6 +4,6 @@ draft: false
 weight: 2
 ---
 
-This section contains information on how to bootstrap a minimal cluster with Boundless. Additional setups can be found in the [examples](/docs/examples) section.
+This section contains information on how to bootstrap a minimal cluster with Boundless. Additional setups can be found in the [examples](../examples) section.
```

Comment:
We'll want to use `/boundless/docs/examples` as the URL is appended to the hostname. This might work on local systems but I believe this would be a broken link once it is deployed to github.

Reply:
As per GitHub relative link docs, all we should be doing is to put the relative links and it's supposed to work fine. What my change does is to fix the relative link as `/docs/install` e.g. is not a relative link from the file where it's referencing it. I think it should work fine.

Reply:
see https://github.blog/2013-01-31-relative-links-in-markup-files/

Reply:
For these markdown files, it's actually [Hugo](https://gohugo.io/) that is parsing them and determining the final path.

I generated the published output on my local machine and the links look like they are correct. We can put this change in and once I take a look as the BaseURL in the config, I can change them over if we need to.